File: BOJ/수학/23971.py
# 23971 ZOAC 4
# https://www.acmicpc.net/problem/23971
import sys
import math
input = sys.stdin.readline

H, W, N, M = map(int, input().split())

# 앉을 수 있는 자리 개수 = [ 길이 / ( 비우고 앉아야 하는 자리 개수 + 1) ] 을 반올림 한 값
print(math.ceil(H/(N+1))*math.ceil(W/(M+1)))

File: BOJ/수학/23971/23971.cpp
// 23971 ZOAC 4
// https://www.acmicpc.net/problem/23971
#include <iostream>
#include <cmath>

using namespace std;

int main(void) {
  ios_base::sync_with_stdio(false); 
  cin.tie(NULL); 
  cout.tie(NULL);

  int H, W, N, M;
  cin >> H >> W >> N >> M;
  double a = (double)H / (double(N)+1);
  double b = (double)W / (double(M)+1);
  cout << (int)ceil(a) * (int)ceil(b);

  return 0;
}